decidim-core/app/views/decidim/messaging/conversations/show.html.erb

```diff
@@ -5,7 +5,9 @@
     <% end %>
   </div>
 
-  <% if conversation.accept_user?(current_user) %>
+  <% if conversation.with_deleted_users?(current_user) %>
+    <em><%= t "decidim.user_conversations.show.deleted_accounts" %></em>
```

Comment:
The same regarding the styling. Also, for consistency, we should use the same i18n format as the rest (you'll need to add it in the `en.yml` file, in `en.decidim.messaging.conversations.show.deleted_accounts`)
```suggestion
    <div class="callout warning margin-top-2"><%= t ".deleted_accounts" %></div>
```